peps/pep-0723.rst

```diff
@@ -16,12 +16,6 @@ Post-History: `04-Aug-2023 <https://discuss.python.org/t/30979>`__,
 Replaces: 722
 Resolution: https://discuss.python.org/t/40418/82
 
```

Comment:
You'll need to add a target to https://packaging.python.org/en/latest/specifications/inline-script-metadata/ first, I believe.

```suggestion

.. canonical-pypa-spec:: :ref:`packaging:inline-script-metadata`


```

Reply:
How about `` :external+packaging:doc:`specifications/inline-script-metadata` ``?

Reply:
I prefer we link to the specific ref, so that the PEPs repository isn't coupled with the specific layout of packaging.python.org's documents.